src/pages/Layout/Layout.js

```diff
@@ -1,20 +1,19 @@
 import { Outlet } from "react-router-dom";
 import { Link } from "./Link/Link";
+import './Link/Link.scss'
 
 
 
 const Layout = () => {
   return (
     <>
-      <nav>
-        <ul>
-          <li>
-            <Link to="/">Хто ми</Link>
-          </li>
-          <li>
-            <Link to="/app">Blogs</Link>
-          </li>
-        </ul>
+      <nav  className="naw-wrapper">
```

Comment:
It's not a nav wrapper, it's the nav itself :)
I suggest renaming `naw-wrapper` -> `nav`, or `navbar`